website/docs/d/push_based_log_export.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
is this generated automatically from the templates?

Reply:
yes